exporter/otlphttpexporter/otlp.go

```diff
@@ -270,12 +270,17 @@ func tracesPartialSuccessHandler(protoBytes []byte, contentType string) error {
 	}
 	partialSuccess := exportResponse.PartialSuccess()
 	if !(partialSuccess.ErrorMessage() == "" && partialSuccess.RejectedSpans() == 0) {
-		return consumererror.NewPermanent(fmt.Errorf("OTLP partial success: %s (%d rejected)", partialSuccess.ErrorMessage(), partialSuccess.RejectedSpans()))
+		// TODO: These should be counted as "rejected", similar to dropped items.
+		// https://github.com/open-telemetry/opentelemetry-collector/issues/9243
+		e.logger.Warn("partial success",
+			zap.String("message", exportResponse.PartialSuccess().ErrorMessage()),
+			zap.Int64("num_rejected", exportResponse.PartialSuccess().RejectedSpans()),
```

Comment:
These should be updated as well right?

Reply:
I'm pretty confused.  The commit https://github.com/open-telemetry/opentelemetry-collector/pull/9260/commits/7920e791f80f8b7658cc00b9616e5f791307e021 shows I changed these lines, followed by a clean merge commit, and my local copy was different than what you saw in the review.

I rebased and force-pushed, now the fix I had intended appears.

Reply:
I was confused between the two otlp.go files (the one HTTP and the other gRPC).